src/lib.rs

```diff
@@ -658,15 +674,25 @@ impl FromData for TableRecord {
 }
 
 #[cfg(feature = "variable-fonts")]
-const MAX_VAR_COORDS: usize = 32;
+const MAX_VAR_COORDS: usize = 64;
 
 #[cfg(feature = "variable-fonts")]
-#[derive(Clone, Default)]
+#[derive(Clone)]
 struct VarCoords {
     data: [NormalizedCoordinate; MAX_VAR_COORDS],
     len: u8,
 }
 
+#[cfg(feature = "variable-fonts")]
```

Comment:
Why not derive?

Reply:
It tells me it's not implemented for `[NormalizedCoordinate; 64]`... Looks like it only works up to a certain number.

Reply:
Got it. Probably limited to hardcoded value.